### PR TITLE
Update / refactor the Ohai docs a bit

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: chef_master/source/chef_client.rst [hunk 1]
```diff
@@ -57,20 +57,20 @@ The key components of nodes that are under management by Chef include:
...
 
-       The types of attributes Ohai collects include (but are not limited to):
+       The types of attributes Ohai collects include but are not limited to:
 
-       * Platform details
-       * Network usage
-       * Memory usage
-       * CPU data
-       * Kernel data
+       * Platform
```
Comment (thommay) [inline chef_master/source/chef_client.rst]: As a casual reader, I can't really tell what `platform` might mean here. I think this bullet might be better written as "Operating System details"

File: chef_master/source/ohai.rst [hunk 2]
```diff
@@ -5,26 +5,26 @@ About Ohai
 
 .. tag ohai_summary
 
-Ohai is a tool that is used to detect attributes on a node, and then provide these attributes to the chef-client at the start of every chef-client run. Ohai is required by the chef-client and must be present on a node. (Ohai is installed on a node as part of the chef-client install process.)
+Ohai is a tool that is used to collect system configuration data, which is provided to the chef-client for use within cookbooks. Ohai is run by the chef-client at the beginning of every Chef run to determine system state. It includes multiple built in plugins for configuration detection and a plugin model for writing custom plugins.
```
Comment (thommay) [inline chef_master/source/ohai.rst]: "includes many built in plugins" ?

File: chef_master/source/ohai_custom.rst [hunk 3]
```diff
@@ -434,7 +410,7 @@ and so on, for each of the various cloud providers.
 
 .. end_tag
 
-Log Entries
+Loggging
```
Comment (thommay) [inline chef_master/source/ohai_custom.rst]: i like your commitment to the letter 'g' here

**Review by thommay — CHANGES_REQUESTED** [review]

File: chef_master/source/ohai_custom.rst (hunk 3 above)
Comment (tas50) [reply chef_master/source/ohai_custom.rst]: We loggg a lot

File: chef_master/source/ohai.rst (hunk 2 above)
Comment (tas50) [reply chef_master/source/ohai.rst]: The updated wording work for you?

File: chef_master/source/chef_client.rst (hunk 1 above)
Comment (tas50) [reply chef_master/source/chef_client.rst]: Changed to Operating System

**Review by thommay — APPROVED** [review]

**Comment by tas50** [comment]
@davidwrede This is ready for merge now

**Comment by kagarmoe** [comment]
tenor-12262506

**Pull request merged**